package-lock.json

```diff
@@ -7163,305 +7163,305 @@
-        "@tradeshift/elements.icon": "^0.34.0",
-        "@tradeshift/elements.overlay": "^0.34.0",
-        "@tradeshift/elements.select-menu": "^0.34.0"
+        "@tradeshift/elements": "^0.35.0",
+        "@tradeshift/elements.icon": "^0.35.0",
+        "@tradeshift/elements.overlay": "^0.35.0",
+        "@tradeshift/elements.select-menu": "^0.35.0"
       }
     },
     "@tradeshift/elements.select": {
       "version": "file:packages/components/select",
       "requires": {
-        "@tradeshift/elements": "^0.34.0",
-        "@tradeshift/elements.icon": "^0.34.0",
-        "@tradeshift/elements.overlay": "^0.34.0",
-        "@tradeshift/elements.select-menu": "^0.34.0"
+        "@tradeshift/elements": "^0.35.0",
+        "@tradeshift/elements.icon": "^0.35.0",
+        "@tradeshift/elements.overlay": "^0.35.0",
+        "@tradeshift/elements.select-menu": "^0.35.0"
       }
     },
     "@tradeshift/elements.select-menu": {
       "version": "file:packages/components/select-menu",
       "requires": {
-        "@tradeshift/elements": "^0.34.0",
-        "@tradeshift/elements.button": "^0.34.0",
-        "@tradeshift/elements.button-group": "^0.34.0",
-        "@tradeshift/elements.icon": "^0.34.0",
-        "@tradeshift/elements.list-item": "^0.34.0",
-        "@tradeshift/elements.spinner": "^0.34.0"
+        "@tradeshift/elements": "^0.35.0",
+        "@tradeshift/elements.button": "^0.35.0",
+        "@tradeshift/elements.button-group": "^0.35.0",
+        "@tradeshift/elements.icon": "^0.35.0",
+        "@tradeshift/elements.list-item": "^0.35.0",
+        "@tradeshift/elements.spinner": "^0.35.0"
       }
     },
     "@tradeshift/elements.spinner": {
       "version": "file:packages/components/spinner",
       "requires": {
-        "@tradeshift/elements": "^0.34.0"
+        "@tradeshift/elements": "^0.35.0"
       }
     },
     "@tradeshift/elements.status": {
       "version": "file:packages/components/status",
       "requires": {
-        "@tradeshift/elements": "^0.34.0"
+        "@tradeshift/elements": "^0.35.0"
       }
     },
     "@tradeshift/elements.tab": {
       "version": "file:packages/components/tab",
       "requires": {
-        "@tradeshift/elements": "^0.34.0"
+        "@tradeshift/elements": "^0.35.0"
       }
     },
     "@tradeshift/elements.tabs": {
       "version": "file:packages/components/tabs",
       "requires": {
-        "@tradeshift/elements": "^0.34.0",
-        "@tradeshift/elements.icon": "^0.34.0",
-        "@tradeshift/elements.tab": "^0.34.0",
-        "@tradeshift/elements.typography": "^0.34.0"
+        "@tradeshift/elements": "^0.35.0",
+        "@tradeshift/elements.icon": "^0.35.0",
+        "@tradeshift/elements.tab": "^0.35.0",
+        "@tradeshift/elements.typography": "^0.35.0"
       }
     },
     "@tradeshift/elements.tag": {
       "version": "file:packages/components/tag",
       "requires": {
         "@tradeshift/elements": "^0.33.8",
```

Comment:
Hmm, why this dependency is not updated?

Reply:
Super catch. I haven't analysed the version creating script, but I guess because it was never released.

Reply:
I think it should be fixed here: https://github.com/Tradeshift/elements/blob/master/packages/components/tag/package.json

Reply:
Should I just manually make it 0.35.0?